<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>picv</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ImageViewer" name="imageViewer"/>
    </item>
    <item>
     <widget class="ImageList" name="imageList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="gridSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenPath"/>
    <addaction name="actionReloadPath"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFit"/>
    <addaction name="actionRotateRight"/>
    <addaction name="actionRotateLeft"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenPath">
   <property name="text">
    <string>Open Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionReloadPath">
   <property name="text">
    <string>Reload Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="text">
    <string>Fit</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="text">
    <string>RotateRight</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="text">
    <string>RotateLeft</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>.image_viewer</header>
  </customwidget>
  <customwidget>
   <class>ImageList</class>
   <extends>QListWidget</extends>
   <header>.image_list</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
